packages/odyssey-react/src/components/Status/Status.module.scss

```diff
@@ -34,26 +34,26 @@ $status-indicator-size: 1em * 0.5;
     height: $status-indicator-size;
     margin-inline-end: $spacing-xs;
     transform: translateY(-50%);
-    background-color: cv("gray", "600");
+    background-color: var(--ColorIndicatorNeutral);
```

Comment:
dang naming is hard, I like the name you have here as it clearly describes what the the custom prop controls, but it doesn't [align 100% with the naming scheme here](https://docs.google.com/document/d/1ZXQahqi_ihNnHIGqE_VoruOzt5CVWmgnJq2_niHI51w/edit) based on my understanding.

For example, if we name this `ColorBackgroundIndicatorNeutral` would it be more correct? Either way is fine with me. cc @arnoldsandoval-okta

Reply:
Ah, I read the convention scopes as "use" rather than "property". Here, it's implemented as a background color, but isn't intended to have content over it, so "Indicator" felt like a more appropriate scope.

Interested in Arnie's thoughts as well.

Reply:
This is correct, scope refers to use (typically components, or a grouping of components - in this case, `Indicator`)